<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diaphragm Forces</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../civiltools.qrc">
    <normaloff>:/civiltools/images/property_editor.svg</normaloff>:/civiltools/images/property_editor.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="stories">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="collector_groupbox">
         <property name="title">
          <string>Collector</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Omega x</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="omega_x">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Omega x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="omega_y">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Horizontal Irregularity</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="reentrance_corner_checkbox">
            <property name="text">
             <string>Reentrant Corner</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="diaphragm_discontinuity_checkbox">
            <property name="text">
             <string>Diaphragm Discontinuity</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="out_of_plane_offset_checkbox">
            <property name="text">
             <string>Out-of-Plane Offset</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Vertical Irregularity</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QCheckBox" name="in_plane_discontinuity_checkbox">
            <property name="text">
             <string>In-Plane Discontinuity</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string notr="true">Hazard Zone</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="risk_level">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>خیلی زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>زیاد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>متوسط</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>کم</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string notr="true">Importance Factor</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="Gui::PrefComboBox" name="importance_factor">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <property name="prefEntry" stdset="0">
             <cstring>importance_factor</cstring>
            </property>
            <property name="prefPath" stdset="0">
             <cstring>Mod/civilTools</cstring>
            </property>
            <item>
             <property name="text">
              <string>0.8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.4</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="create_pushbutton">
     <property name="text">
      <string>Create</string>
     </property>
     <property name="icon">
      <iconset resource="../../civiltools.qrc">
       <normaloff>:/civiltools/images/etabs.png</normaloff>:/civiltools/images/etabs.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefComboBox</class>
   <extends>QComboBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../civiltools.qrc"/>
 </resources>
 <connections/>
</ui>
